tests/microos/rebuild_initrd.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Rebuild initrd without ignition and combustion
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use transactional;
+
+sub run {
+    my ($self) = @_;
+    select_console 'root-console';
+
+    assert_script_run(q|echo 'omit_dracutmodules+="ignition ignition-microos combustion"' > /etc/dracut.conf.d/20-disable_ignition.conf|);
```

Comment:
interesting usage of `q||`, never seen it before, I've always used `q()` or `qq()` ... 
 https://perldoc.perl.org/perlop#Quote-and-Quote-like-Operators

Reply:
Yeah, you can use whatever
```
In the following table, a {} represents any pair of delimiters you choose.
```

In the code base also have `@` for instance:

```
ests/sles4sap/hana_test.pm:        assert_script_run q@df -k $(awk '/nfs/ {print $2}' /etc/fstab)@;
```

Reply:
nice perl, nice! ...